staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/gvkparser.go

```diff
@@ -55,11 +55,15 @@ func newGVKParser(models proto.Models) (*gvkParser, error) {
 	for _, modelName := range models.ListModels() {
 		model := models.LookupModel(modelName)
 		if model == nil {
-			panic("ListModels returns a model that can't be looked-up.")
+			panic(fmt.Sprintf("ListModels returns a model that can't be looked-up for: %v", modelName))
```

Comment:
Do we want to keep the panic here as well?
cc. @apelisse / @jennybuckley

Reply:
Yeah, this is not an error, this is an assertion.